dataactvalidator/app.py

```diff
@@ -39,22 +42,39 @@ def before_request():
     def handle_response_exception(error):
         """Handle exceptions explicitly raised during validation."""
         logger.error(str(error))
+        job_id = g.get('job_id', None)
```

Comment:
Fair point. It seems safe enough, since this app only has one functional route, and the beginning of that route's code is putting the `job_id` in flask.g. I'll add some comments to this effect--good idea.